src/Builder/FormContractor.php

```diff
@@ -197,15 +194,16 @@ private function hasAssociation(FieldDescriptionInterface $fieldDescription)
     }
 
     /**
-     * @param string $type
-     * @param array  $classes
-     *
-     * @return array
+     * @param string[] $classes
      */
-    private function checkFormClass($type, $classes)
+    private function checkFormClass(string $type, array $classes): bool
```

Comment:
Maybe we could add a description explaining the purpose of this method and how it behaves with valid and invalid values.

Reply:
This method just check if the type is one of the classes provided.

A right name for this method should be enough IMHO.

Reply:
`private function isInstanceOf(string $type, array $classes): bool` @phansys ?

Reply:
I'm not convinced since the check will be performed against multiple values and it will return `true` if at least 1 matches, but I can't provide a better name.
My best effort: `containsInstanceOf()`.

Reply:
The method check if `$type` is instance of one of the class provided.
`isInstanceOfOneOf` is more explicit, maybe ?

I'm not convinced by `containsInstanceOf`.
Do you mean that `$classes` contains an instance of `$type` ?

Reply:
> The method check if `$type` is instance of one of the class provided.
> `isInstanceOfOneOf` is more explicit, maybe ?

Maybe `isAnyInstanceOf()` / `isTypeInArray()`? We could try to find inspiration at PHPUnit assertions.

> Do you mean that `$classes` contains an instance of `$type` ?

Yes, exactly.

Reply:
I like `isAnyInstanceOf`. I made the change.